package.json

```diff
@@ -13,10 +13,8 @@
   },
   "workspaces": {
     "packages": [
-      "packages/*"
-    ],
-    "nohoist": [
-      "**"
+      "packages/*",
+      "../times-components/packages/*"
```

Comment:
This line is quite experimental and only needed during development. Having times components in workspace configuration enables us to utilize all the symlinking magic by yarn. However, doing this inclusion with git submodules or os level symlinks might be a better approach.

Reply:
+1 for gitmodules